site/gatsby-config.js

```diff
@@ -14,6 +14,13 @@ module.exports = {
       },
     ],
   },
+  flags: {
+    DEV_SSR: true,
+    FAST_DEV: true,
+    FAST_REFRESH: true,
```

Comment:
👍

Reply:
Doesn't seem to make anything faster though.

Reply:
Reverted :laughing: